pkg/build/gobuild.go

```diff
@@ -86,19 +86,41 @@ func NewGo(options ...Option) (Interface, error) {
 	return gbo.Open()
 }
 
+// findImportPath uses the go list command to find the full import path from a package.
+func (g *gobuild) findImportPath(importPath, format string) (string, error) {
+	cmd := exec.Command("go", "list", "-f", "{{."+format+"}}", importPath)
+	stdout, err := cmd.Output()
+	if err != nil {
+		return "", err
+	}
+
+	outputs := strings.Split(string(stdout), "\n")
+	if len(outputs) == 0 {
+		return "", errors.New("could not find specified import path: " + importPath)
+	}
+
+	output := outputs[0]
+	if output == "" {
+		return "", errors.New("import path is ambiguous: " + importPath)
+	}
+
+	return output, nil
+}
+
 // IsSupportedReference implements build.Interface
 //
 // Only valid importpaths that provide commands (i.e., are "package main") are
 // supported.
-func (*gobuild) IsSupportedReference(s string) bool {
-	p, err := gb.Import(s, gb.Default.GOPATH, gb.ImportComment)
+func (g *gobuild) IsSupportedReference(s string) bool {
+	output, err := g.findImportPath(s, "Name")
```

Comment:
what is `"Name"` ?

Reply:
`Name` is the Package's Name from a given import as can seen (on the docs)[https://github.com/golang/tools/blob/master/go/packages/packages.go#